webrtc/receiver-track-live.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Remote tracks should not get ended except for stop/close</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="RTCPeerConnection-helper.js"></script>
+</head>
+<body>
+    <video id="video" controls autoplay playsinline></video>
+    <script>
+    var pc1, pc2;
+    var localTrack, remoteTrack;
+    promise_test(async (test) => {
+        const localStream = await navigator.mediaDevices.getUserMedia({audio: true});
```

Comment:
Maybe use [getNoiseStream](https://github.com/web-platform-tests/wpt/blob/master/webrtc/RTCPeerConnection-helper.js#L531)?

Reply:
I prefer getUserMedia and this is used in other tests.